src/Plugin/Block/AlertBannerBlock.php

```diff
@@ -65,6 +66,46 @@ public static function create(ContainerInterface $container, array $configuratio
     );
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultConfiguration() {
+    return [
+      'include_types' => [],
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function blockForm($form, FormStateInterface $form_state) : array {
+    $form = parent::blockForm($form, $form_state);
+    $type_storage = $this->entityTypeManager->getStorage('localgov_alert_banner_type');
+    $config = $this->getConfiguration();
+    $config_options = [];
+    foreach ($type_storage->loadMultiple() as $id => $type) {
+      $config_options[$id] = $type->label();
+    }
+    $form['include_types'] = [
+      '#type' => 'checkboxes',
+      '#options' => $config_options,
+      '#title' => $this->t('Display types'),
+      '#description' => $this->t('If no types are selected all will be displayed.'),
+      '#default_value' => !empty($config['include_types']) ? $config['include_types'] : '',
```

Comment:
@finnlewis @iam-jam I think this might be what is causing the error.
I think default value should be an array (even if it is null).

Reply:
Yep that's it @andybroomfield. I had actually spotted this before but forgot to fix it. Shame this hasn't been picked up in tests.

Reply:
Just to confirm that changing 
`'#default_value' => !empty($config['include_types']) ? $config['include_types'] : '',`
to
`'#default_value' => !empty($config['include_types']) ? $config['include_types'] : [],`
resolves this issue. If we can get this change added we can get this in a new release.

Reply:
Ditto. @j4-m could you commit the change to your forked branch?